lib/roo/excelx/sheet_doc.rb

```diff
@@ -207,7 +207,7 @@ def expand_merged_ranges(cells)
         end
         # Duplicate value into all cells in merged range
         merges.each do |dst, src|
-          cells[dst] = cells[src]
+          cells[dst] = cells[src] if cells[src]
```

Comment:
I think we should add this check before adding  range to merges hash. Just after https://github.com/ttilberg/roo/blob/e46239acbde159ac1b63de681450a54ffbbb31e2/lib/roo/excelx/sheet_doc.rb#L200 seems like right place to add this check

Reply:
Good idea. I was having a hard time parsing the above code (the `tl` `br`), but you're right. If `cells[tl]` is nil, then none of it's iterations should get collected, potentially saving some work. I updated the variable names to help make a bit more clear what was going on -- I didn't see that `tl` and `br` were commonly used elsewhere. If these have more context than I know of, I'm happy to change them back.

Thanks!